tests/Select.test.tsx

```diff
@@ -1949,4 +1949,26 @@ describe('Select.Basic', () => {
       expect(wrapper.find('span.rc-select-selection-item').at(2).prop('title')).toEqual('+ 1 ...');
     });
   });
+
+  it('should be focused when click clear button', () => {
+    jest.useFakeTimers();
+
+    const mouseDownPreventDefault = jest.fn();
+    const wrapper = mount(
+      <Select allowClear value={1}>
+        <Option value="1">1</Option>
+        <Option value="2">2</Option>
+      </Select>,
+    );
```

Comment:
should attchTo body since enzyme not fire `onFocus` if node is not in the document